test/e2e/prune_test.go

```diff
@@ -208,6 +209,7 @@ var _ = Describe("Podman prune", func() {
 
 	It("podman system prune pods", func() {
 		useCustomNetworkDir(podmanTest, tempdir)
+		// FIXME: this is not used by anything. Does it serve a purpose?
```

Comment:
The point is to have two pods but the created one should not be pruned only the stopped one.

Reply:
And if I had read the whole test I would've seen that. That was careless of me.